recordingclient.go

```diff
@@ -0,0 +1,53 @@
+package lksdk
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/livekit/protocol/auth"
+	"github.com/twitchtv/twirp"
+
+	livekit "github.com/livekit/server-sdk-go/proto"
+)
+
+type RecordingServiceClient struct {
+	livekit.RecordingService
+	apiKey    string
+	apiSecret string
+}
+
+func NewRecordingServiceClient(url string, apiKey string, secretKey string) *RecordingServiceClient {
+	client := livekit.NewRecordingServiceProtobufClient(url, &http.Client{})
+	return &RecordingServiceClient{
+		RecordingService: client,
+		apiKey:           apiKey,
+		apiSecret:        secretKey,
+	}
+}
+
+func (c *RecordingServiceClient) StartRecording(ctx context.Context, req *livekit.StartRecordingRequest) (*livekit.RecordingResponse, error) {
+	ctx, err := c.withAuth(ctx, auth.VideoGrant{RoomAdmin: true})
```

Comment:
similar comment as in server. currently roomAdmin requires a room. let's chat how we'll want to treat this permission.